File: problems/Two City Scheduling/twoCitySchedCost.java
class Solution {
    public int twoCitySchedCost(int[][] costs) {
        int result = 0;
        int n = costs.length / 2;

        Arrays.sort(costs, (a, b) ->
            (a[0] - a[1]) - (b[0] - b[1])
        );

        for (int idx = 0; n > idx; ++idx) {
            result += costs[idx][0] + costs[n + idx][1];
        }

        return result;
    }
}

File: problems/Two City Scheduling/twoCitySchedCost.js
/**
 * @param {number[][]} costs
 * @return {number}
 */
var twoCitySchedCost = function(costs) {
    const n = Math.trunc(costs.length/2);
    let result = 0;

    costs.sort((a,b)=>a[0]-a[1]-(b[0]-b[1]));

    for (let idx = 0; n > idx; ++idx) {
        result += costs[idx][0] + costs[n + idx][1];
    }

    return result;
};
